### ISSUE TITLE
[Rails 5] Replace db2fog with backup gem

### ISSUE THREAD

**Issue by Matt-Yorkley** [openfoodfoundation/openfoodnetwork#6803]
### What we should change and why (this is tech debt)

The db2fog gem is not maintained and is not compatible with newer versions of Rails, including 5.0.

The [backup](https://github.com/backup/backup) gem seems to be the closest and most widely-used equivalent.

### Context

Rails 5 upgrade.

### Impact and timeline

Needs to be done before we can switch to Rails 5.

**Comment by luisramos0** [comment]
yes :+1: duplicate of of https://github.com/openfoodfoundation/openfoodnetwork/issues/6669

fyi there's this epic to manage all rails 5 things: https://github.com/openfoodfoundation/openfoodnetwork/issues/6346

**Comment by Matt-Yorkley** [comment]
It turns out the backup gem breaks our S3 images setup, and it also means we would lose some functionality in restoring backups.

Also; turns out the good old `db2fog` gem actually _does_ work with Rails 5, it just needed the dependencies loosening a bit to allow it to be installed with bundler.
